client/src/components/EventButton.js

```diff
@@ -18,7 +18,7 @@ position:fixed;
 height:50px;
 width:50px;
```

Comment:
You shouldnt set a fix width and height for a button. Imagine your text inside the button has to be very long, then you couldnt use the button because the text gets cut off.
I would set a padding to the button, so it looks nice, but let the content decide how big the button should be.